app/services/hyku_addons/validations/solr_entry_validation_service.rb

```diff
@@ -0,0 +1,183 @@
+# frozen_string_literal: true
+
+module HykuAddons
+  module Validations
+    class SolrEntryValidationService < EntryValidationService
+      attr_reader :errors, :entry
+
+      SOURCE_SERVICE_OPTIONS = {
+        base_url: ENV['BULKRAX_SOURCE_BASE_URL'],
+        username: ENV['BULKRAX_SOURCE_USERNAME'],
+        password: ENV['BULKRAX_SOURCE_PASSWORD']
+      }.with_indifferent_access.freeze
+
+      DESTINATION_SERVICE_OPTIONS = {
+        base_url: ENV['BULKRAX_DESTINATION_BASE_URL'],
+        username: ENV['BULKRAX_DESTINATION_USERNAME'],
+        password: ENV['BULKRAX_DESTINATION_PASSWORD']
+      }.with_indifferent_access.freeze
+
+      EXCLUDED_FIELDS = %i[
```

Comment:
A lot of the values and logic in this file were probably specific to the hyku1 migration we did so maybe it makes sense to rename this or slowly refine those idiosyncrasies away over time (or into a specific subclass if we want to keep them around).  Nothing needs to be done for this PR but just thoughts for later.